shotover-proxy/src/transforms/mod.rs

```diff
@@ -105,6 +107,7 @@ impl Transforms {
             Transforms::Tee(m) => m.transform(message_wrapper).await,
             Transforms::DebugPrinter(p) => p.transform(message_wrapper).await,
             Transforms::Null(n) => n.transform(message_wrapper).await,
+            #[cfg(test)]
```

Comment:
Oh wait, cant we get away with just adding the `#[cfg(test)]` to the `TransformsConfig::get_transforms` Loopback arm?

Reply:
Yes. Though only compiling/including what we need in our build would be a good habit to follow.

Reply:
Is the advantage of that a smaller binary size?
I think in general we want to keep our test and production binaries as close as possible.
But in this particular case im not super worried either way, so just resolve this if you'd rather leave it as is.

Reply:
Yes I was thinking smaller binary size.